### ISSUE TITLE
Subfolder does not display content, but shows 404 when viewed from nested page

### ISSUE THREAD

**Issue by mccurcio** [vsoch/docsy-jekyll#58]
I am having trouble viewing subfolders after I have drilled down to content.
For example, when I am viewing [](https://vsoch.github.io/docsy-jekyll/docs/subfolder/example-page) there is the pseudo-path above it, [Documentation / Subfolder / A Nested Page]. If I want to open the subfolder [](https://vsoch.github.io/docsy-jekyll/docs/subfolder/) I get a 404 error and not the content of the subfolder as I would expect. 
I would like to have my folder open with the contents showing, similar to the document page that shows the md files. Originally I added an 'index.md' file into each folder with a re-direct sending the viewer back to the home/index page. This did not work. 
Is there something I am missing or is this an issue?
Thank you
M

**Comment by vsoch** [comment]
You're totally right! Check out this example https://vsoch.github.io/docsy-jekyll/docs/subfolder rendered from https://github.com/vsoch/docsy-jekyll/blob/master/_docs/subfolder.md. In summary - you name a markdown file the "same" as the folder (and at the same root) for it to appear as an index for that page/url.

**Comment by mccurcio** [comment]
Hi V,
I'm sorry, I still can't get it to work for me. 
- On your example site moving from `docs/subfolder/example-page`  using the links on the top where it says `/subfolder/` still gets me a 404. Trying to get to  `docs/subfolder/`.
- Then on my site, after placing markdown files at the same root as the folder name gives 404. It does not allow moving upper a directory. 
- I am puzzled.

**Comment by vsoch** [comment]
The pages are there:

 - https://vsoch.github.io/docsy-jekyll/docs/subfolder
 - https://vsoch.github.io/docsy-jekyll/docs/subfolder/example-page
 
As for the breadcrumb It's likely the trailing slash (try removing it). Some browsers will treat them identically (e.g., subfolder/ vs subfolder) but others do not. If you'd like to submit a PR to remove the slash from the breadcrumb I'd be happy to review.

**Comment by mccurcio** [comment]
Hi V,
Thanks for the offer. 
Let me play around with this first. I'm not sure it is a slash issue but let me convince myself either way. 
Can I keep this open until then? ;^)

**Comment by vsoch** [comment]
Yep sure thing. If it helps look in the _site folder to see the raw files that are rendered.

**Comment by vsoch** [comment]
Yes I'd love to support you to do that! I'll add the topic to the repository.
